styles/base.css

```diff
@@ -0,0 +1,31 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-size: 1em;
+  font-weight: normal;
+  font-style: normal;
+  width: 100%;
```

Comment:
This will have unintended consequences.   It is best to avoid using width 100% as this causes inline, inline-block, and block elements to all behave as block elements.